demo/src/app/guides/custom-formly-field.md

```diff
@@ -101,3 +101,27 @@ The live example can be found in stackblitz: [demo](https://stackblitz.com/edit/
       ];
     }
     ```
+
+## Troubleshooting
```

Comment:
Move this into a separate file `troubleshooting.md`, also it would be better to add a doc for `FormlySelectModule` module 🙏.